src/lib.rs

```diff
@@ -9,7 +52,7 @@ use proc_macro::TokenStream;
 ///
 /// Structure:
 ///
-/// ```no_test
+/// ```ignore
```

Comment:
```suggestion
/// ```no_run
/// # #[cfg(doc)]
```

Reply:
This can avoid the warning, but also remove the info that this code cannot be compiled. This code has placeholder types like `type1`.